src/backend/libs/justin-data-starter/pom.xml

```diff
@@ -123,7 +123,7 @@
 			<plugin>
 				<groupId>org.codehaus.mojo</groupId>
 				<artifactId>build-helper-maven-plugin</artifactId>
-				<version>1.8</version>
+				<version>3.2.0</version>
```

Comment:
this is a major version update, not sure if we want this @TayGov

Reply:
No that is a huge bump. Might be something we want to test locally first.